data/mods/aftershock_exoplanet/region_settings.json

```diff
@@ -35,7 +35,16 @@
       "park_sigma": 80,
       "houses": { "afs_city_ruinfield": 400, "afs_formless_ruins_dynamic": 600, "afs_house_1": 300 },
       "parks": { "afs_city_ruinfield": 100 },
-      "shops": { "afs_augmentation_clinic_1": 400, "afs_astrobiology_lab": 400 }
+      "shops": {
+        "afs_augmentation_clinic_1": 400,
+        "city_block_2": 700,
+        "urban_35_hospital": 875,
+        "office_skyscraper": 400,
+        "office_tower_large": 400,
+        "office_tower_hiddenlab": 800,
+        "afs_astrobiology_lab": 300,
+        "mall": 700
+      }
```

Comment:
I think these changes got added here by mistake, you should keep the orignial list.

Should be ready for merging after that.

Reply:
I have restored the points you raised and would appreciate your confirmation.